dsls/sql/src/main/java/org/apache/beam/dsls/sql/interpreter/operator/string/BeamSqlTrimExpression.java

```diff
@@ -63,16 +63,16 @@ public BeamSqlTrimExpression(List<BeamSqlExpression> operands) {
       return BeamSqlPrimitive.of(SqlTypeName.VARCHAR,
           opValueEvaluated(0, inputRecord).toString().trim());
     } else {
-      String type = opValueEvaluated(0, inputRecord);
+      SqlTrimFunction.Flag type = opValueEvaluated(0, inputRecord);
```

Comment:
Don't think so, previously I assume we can treat the `LEADING`, `TRAILING` etc as a string, but actually they are not, they are flags.